src/modules/launcher/Plugins/Microsoft.Plugin.Program/Storage/PackageRepository.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Plugin.Program.Logger;
+using Microsoft.Plugin.Program.Programs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel;
+using Wox.Infrastructure.Storage;
+
+namespace Microsoft.Plugin.Program.Storage
+{
+    /// <summary>
+    /// A repository for storing packaged applications such as UWP apps or appx packaged desktop apps.
+    /// This repository will also monitor for changes to the PackageCatelog and update the repository accordingly
+    /// </summary>
+    internal class PackageRepository : ListRepository<UWP.Application>, IRepository<UWP.Application>, IProgramRepository

```

Comment:
[nit] `IRepository<UWP.Application>` can be removed here as it is already inherited by `ListRepository`

Reply:
Done